integration/meta-test.ts

```diff
@@ -52,29 +56,34 @@ describe("meta", () => {
     app = await createAppFixture(fixture);
   });
 
-  afterAll(async () => {
-    await app.close();
-  });
+  afterAll(async () => await app.close());
 
-  test("meta { title } adds a <title />", async () => {
+  test("empty meta does not render a tag", async () => {
     let enableJavaScript = await app.disableJavaScript();
 
     await app.goto("/");
 
-    expect(await app.getHtml("title")).toEqual(
-      expect.stringMatching(/<title>/s)
+    expect(await app.getHtml("meta")).not.toEqual(
+      expect.stringMatching(/<meta\s+property="og:type/s)
     );
```

Comment:
this works, but seems overly verbose

```suggestion
    await expect(app.getHtml('meta[property="og:type"]')).rejects.toThrowError(
      "No element matches selector \"meta[property=\"og:type\"]\""
    );
```